src/GuzzleMiddleware.php

```diff
@@ -36,13 +38,16 @@ public function __invoke(callable $handler) : callable
         return function (Request $request, array $options) use ($handler) {
             $start = microtime(true);
             return $handler($request, $options)->then(
-                function (Response $response) use ($request, $start) {
+                function (Response $response) use ($request, $start, $options) {
+                    $path = !empty($options['actionURI']) ? $options['actionURI'] : self::DEFAULT_GUZZLE_MESSAGE;
```

Comment:
My only concern is that the constant name is somewhat misleading. An alternative is maybe to set the path to an empty string or a new value.

Reply:
Empty is always an option but with a string you have the option to filter it out. If you have empty you don't know what the issue was.